WordPress/gutenberg
# Menu.ItemHelpText: better line breaking

Fixes: #66998 

## What?
Changed CSS word-break property from `break-all` to `break-word` in meta fields for better text readability when content wraps.

## Why?
Currently, text in meta fields breaks incorrectly within words (e.g., "white" breaks as "whit e") due to using `break-all`. This creates poor readability, especially for descriptions and long text content. Using `break-word` ensures text breaks at appropriate word boundaries.

## Testing Instructions
1. Add this test code to create a custom post type with meta fields in `functions.php` in your theme:
```php
function init_test_meta_fields() {
   register_post_type(
       'landmark',
       array(
           'label'        => 'Landmark',
           'public'       => true,
           'supports'     => array( 'title', 'editor', 'comments', 'revisions', 'trackbacks', 'author', 'excerpt', 'page-attributes', 'thumbnail', 'custom-fields', 'post-formats' ),
           'has_archive'  => true,
           'show_in_rest' => true,
       ),
   );
   register_meta(
       'post',
       'landmark_description',
       array(
           'object_subtype' => 'landmark',
           'label'          => 'Description',
           'show_in_rest'   => true,
           'single'         => true,
           'type'           => 'string',
           'default'        => 'Inspired by the Eiffel Tower, this red and white tower is one of Tokyo\'s most recognizable landmarks.',
       )
   );
}

add_action( 'init', 'init_test_meta_fields' );
```

2. Create a new Landmark post
3. Check the attributes >> content desctiption field in meta fields
4. Verify text now breaks correctly at word boundaries
5. Test with various content lengths including URLs and long words

![Your paragraph text](https://github.com/user-attachments/assets/8ce0f88f-e68b-4a70-becf-abf9d9659c8b)


## Screenshots 

Before `word-break: break-all;` : 

![1](https://github.com/user-attachments/assets/0cff0389-278a-4668-8157-00a65ff1335e)


After `word-break: break-word;` : 

![2](https://github.com/user-attachments/assets/3a9b417b-7f64-4afd-b0df-e347bdbb8fcc)
